### PR TITLE
Update BlackHole Callback For Cake 3.2.6

### PR DESCRIPTION
**Pull request by justinyost**
Adds improved docblock for blackHole method call.
Adds information about the exception that is included in 3.2.6 when returned.

### CODE REVIEW

**Comment by markstory** [comment]
I fixed up the typos post merge.

**Comment by justinyost** [comment]
Whoops, thank you for catching and fixing them.

File: en/controllers/components/security.rst [hunk 1]
```diff
@@ -72,6 +72,12 @@ The ``$type`` parameter can have the following values:
   error.
 * 'secure' Indicates an SSL method restriction failure.
 
+.. versionadded:: cakephp/cakephp 2.3.6
```
Comment (antograssiot) [inline en/controllers/components/security.rst]: 3.2.6

File: en/controllers/components/security.rst [hunk 2]
```diff
@@ -72,6 +72,12 @@ The ``$type`` parameter can have the following values:
...
+
+    As of v2.3.6 an additional parameter is included in the backHole callback,
```
Comment (antograssiot) [inline en/controllers/components/security.rst]: here also 3.2.6

File: en/controllers/components/security.rst (hunk 1 above)
Comment (justinyost) [inline en/controllers/components/security.rst]: Thanks for the note, they were fixed here: https://github.com/cakephp/docs/commit/8f3c26958663b02066911164c4e51c34117c6dfd

Comment (antograssiot) [inline en/controllers/components/security.rst]: @justinyost yeah sorry I saw it later. I also fix the namespace in 2cc7af8

Comment (justinyost) [inline en/controllers/components/security.rst]: @antograssiot crud, thank you.

**Pull request merged**